neo/SDK/RPC/RpcClient.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC;
+using Neo.SDK.RPC;
+using Neo.SDK.RPC.Model;
+
+namespace Neo.SDK
```

Comment:
We can move it to `Neo.Network.RPC`. And many classes should be moved into correct namespace.

Reply:
Moved👌